### PR TITLE
#6063 infoschema: tables

### LINKED ISSUE #6063: Fill the TABLE_ROWS field of information_schema.TABLES with the information from statistics
We have updated row_count in the statistics. So we could use it to fill the result of INFORMATION_SCHEMA.TABLES.

### PR DESCRIPTION
**Pull request by dreamquster**
fixed #6063:fill table_rows field in information_schema.tables

### CODE REVIEW

File: infoschema/tables.go [hunk 1]
```diff
@@ -638,32 +659,32 @@ func dataForTables(ctx sessionctx.Context, schemas []*model.DBInfo) [][]types.Da
...
-				"",              // CREATE_OPTIONS
-				table.Comment,   // TABLE_COMMENT
+				catalogVal,             // TABLE_CATALOG
+				schema.Name.O,          // TABLE_SCHEMA
+				table.Name.O,           // TABLE_NAME
+				"BASE TABLE",           // TABLE_TYPE
+				"InnoDB",               // ENGINE
+				uint64(10),             // VERSION
+				"Compact",              // ROW_FORMAT
+				tableRowsMap[table.ID], // TABLE_ROWS
```
Comment (zz-jason) [inline infoschema/tables.go]: can we use statistics handle to do this:
```go
// get a statistics handle
statsHandle := domain.GetDomain(ctx).StatsHandle()

// get table row count by table ID
numTableRows := statsHandle.GetTableStats(tableID).Count
```

**Comment by zz-jason** [comment]
@dreamquster you can run `make check` to fix code format

File: infoschema/tables.go (hunk 1 above)
Comment (dreamquster) [reply infoschema/tables.go]: Good, I'll changed it.

Comment (dreamquster) [reply infoschema/tables.go]: @zz-jason ,
The domain package depends on infoschema, statistics and sessionctx.  It's similar that statistics depends on infoschema.  I'd also tried to return StatsHandle from session.Context, but there's still cycle dependency.  How to get tablestats without cycle importing?

**Comment by dreamquster** [comment]
Merge error, Reopen PR

**Pull request closed without merging**